frontend/src/routes.js

```diff
@@ -137,6 +140,27 @@ const routes = [
         requiresAuth: true,
         title: 'Statping - Service Failures',
       }
+    },{
+      path: 'downtimes',
+      component: DashboardDowntimes,
+      meta: {
+          requiresAuth: true,
+          title: 'Statping - Downtimes',
+      }
+    },{
+      path: 'create_downtime',
+      component: EditDowntime,
+      meta: {
+          requiresAuth: true,
+          title: 'Statping - Create Downtime',
+      }
+    },{
+      path: 'edit_downtime/:id',
```

Comment:
is it part of requirement to use `_` in paths? generally we use `/edit-downtime/:id` type of paths right?

Reply:
did this because it was used through the project route